conf/drupal/config/views.view.content.yml

```diff
@@ -4118,17 +4180,15 @@ display:
         groups:
           1: AND
       exposed_form:
-        type: input_required_on_request
+        type: basic
```

Comment:
@beto-aveiga I think you reverted your prev changes.

Reply:
Thanks! Good catch, you are probably right. I will fix that quickly.